### PR TITLE
Remove old tarballs

### PR DESCRIPTION
**Pull request by toabctl**
When generating the .changes file, sort the messages.

### CODE REVIEW

File: renderspec [hunk 1]
```diff
@@ -115,6 +116,18 @@ def _get_spec_version(specfile):
     return list(set(filter(None, version.decode("utf-8").split('\n'))))[0]
 
 
+def _obs_remove_old_version_source(old_spec_version):
+    """after the spec was update and if a new version is available, remove via
```
Comment (dirkmueller) [inline renderspec]: Was updated

File: renderspec [hunk 2]
```diff
@@ -115,6 +116,18 @@ def _get_spec_version(specfile):
...
+    'osc' the old tarball"""
+    if not os.path.exists(OSC_BIN) or not \
+       os.path.isdir(os.path.join(os.getcwd(), '.osc')):
+        return
+    osc_rm = [OSC_BIN, 'rm', '-f', '*-%s.tar.gz' % (old_spec_version)]
```
Comment (dirkmueller) [inline renderspec]: Do we want to hardcode .gz here?

File: renderspec (hunk 1 above)
Comment (toabctl) [reply renderspec]: done

File: renderspec (hunk 2 above)
Comment (toabctl) [reply renderspec]: better now?

**Review by dirkmueller — APPROVED** [review]

**Pull request merged**